advanced_functionality/scikit_bring_your_own/scikit_bring_your_own.ipynb

```diff
@@ -493,6 +499,88 @@
    "source": [
     "sess.delete_endpoint(predictor.endpoint)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run Batch Transform Job\n",
+    "One can use trained model to get inference on large data sets by using [Amazon SageMaker Batch Transform](https://docs.aws.amazon.com/sagemaker/latest/dg/how-it-works-batch.html). A Batch Transform Job takes your input data S3 location and publishes prediction result to specified S3 output folder. Similar to hosting, lets extract inferences for training data to demo Batch Transform mechanism."
```

Comment:
"You can use a trained model to get inference on large data sets by using [Amazon SageMaker Batch Transform](https://docs.aws.amazon.com/sagemaker/latest/dg/how-it-works-batch.html). A batch transform job takes your input data S3 location and outputs the predictions to the specified S3 output folder. Similar to hosting, you can extract inferences for training data to test batch transform."